components/Dialogs/BUILD

```diff
@@ -162,8 +162,13 @@ swift_library(
     ],
     visibility = ["//visibility:private"],
     deps = [
-        ":Theming",
+        ":Dialogs",
```

Comment:
Are these changes required? It looks like the test already exists after merging develop?  If they are, please update the PR title and description.

Reply:
Yes, there are needed for the theming extension tests. The `BUILD` file had some dependencies but was missing the ones in _Beta_.